config/webpack.config.js

```diff
@@ -229,11 +232,6 @@ const configGenerator = (options, apps) => {
     );
 
     baseConfig.plugins.push(new webpack.HashedModuleIdsPlugin());
-    baseConfig.plugins.push(
-      new ManifestPlugin({
-        fileName: 'file-manifest.json',
-      }),
-    );
     baseConfig.mode = 'production';
```

Comment:
To start pulling assets from dev environments, we now have to generate a ` file-manifest.json` there as well.